components/beebole_app/actions/run-report/run-report.mjs

```diff
@@ -0,0 +1,348 @@
+import app from "../../beebole_app.app.mjs";
+
+export default {
+  name: "Run report",
+  description: "Run a report from Beebole",
+  key: "beebole_app-run-report",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    app,
+    begda: {
+      propDefinition: [
+        app,
+        "begda",
+      ],
+    },
+    endda: {
+      propDefinition: [
+        app,
+        "endda",
+      ],
+    },
+    keys: {
+      type: "string[]",
+      label: "Columns",
+      description: "Select the columns to display in the report.",
+      options: [
```

Comment:
Ideally we'd keep these options in a separate file (e.g. `beebole_app/common/constants.mjs`) to keep the action code clean, and import them here